nestkernel/connection_manager.cpp

```diff
@@ -685,6 +685,10 @@ nest::ConnectionManager::increase_connection_count( const thread tid,
     num_connections_[ tid ].resize( syn_id + 1 );
   }
   ++num_connections_[ tid ][ syn_id ];
+  if ( num_connections_[ tid ][ syn_id ] >= 1 << 27 )
```

Comment:
Can the expression (1 << 27) be changed to `MAX_LCID` after merging #1088? See: https://github.com/nest/nest-simulator/pull/1088/files#diff-42615d7378f1abcdbe8ff2af37737101R78 Thanks!

Reply:
Yes, I'll update it once #1088 is merged.

Reply:
Is this comparison strictly safe, i.e., will everything work with 2^27 connections, or is the limit rather 2^27-1?

Reply:
@heplesser You are right, I have changed it to 2^27-1.